alts/shared/utils/asyncssh.py

```diff
@@ -60,30 +70,76 @@ async def get_connection(self):
             password=self.password,
             client_keys=self.client_keys,
             known_hosts=self.known_hosts,
+            ignore_encrypted=self.ignore_encrypted_keys,
+            preferred_auth=self.preferred_auth,
             env=self.env_vars,
         ) as conn:
             yield conn
 
-    def sync_run_command(self, command: str):
-        try:
-            asyncio.run(self.async_run_command(command))
-        except Exception as exc:
-            logging.exception('Cannot execute asyncssh command: %s', command)
-            raise exc
+    def get_process_results(
+        self,
+        result: asyncssh.SSHCompletedProcess,
+    ) -> Tuple[int, str, str]:
```

Comment:
What about creating an alias of the type e.g. command_result = Tuple[int, str, str], and returning some type with more semantic

Reply:
I have already prepared a `pydantic` model for that in my next commit